src/overlays/actors/ovl_Obj_Switch/z_obj_switch.c

```diff
@@ -382,25 +384,25 @@ void ObjSwitch_FloorUp(ObjSwitch* this, GlobalContext* globalCtx) {
         }
     } else {
         switch ((this->dyna.actor.params >> 4 & 7)) {
-            case OBJSWITCH_SUBTYPE_FLOOR_0:
+            case OBJSWITCH_SUBTYPE_ONCE:
                 if (func_8004356C(&this->dyna)) {
                     ObjSwitch_FloorPressInit(this);
                     ObjSwitch_SetOn(this, globalCtx);
                 }
                 break;
-            case OBJSWITCH_SUBTYPE_FLOOR_1:
+            case OBJSWITCH_SUBTYPE_TOGGLE:
                 if ((this->dyna.unk_160 & 2) && !(this->unk_17F & 2)) {
                     ObjSwitch_FloorPressInit(this);
                     ObjSwitch_SetOn(this, globalCtx);
                 }
                 break;
-            case OBJSWITCH_SUBTYPE_FLOOR_2:
+            case OBJSWITCH_SUBTYPE_RESET:
                 if (func_800435B4(&this->dyna)) {
```

Comment:
"reset" is blue switches right? I think naming it HOLD, after fact that something needs to hold the switch down for it to work would possibly be more clear. but not sure if that works in all contexts

Reply:
yeah it's the blue switches, and yes HOLD seems better than RESET (and I don't think there's a context where it doesn't work since you actually need to have something to keep it held)

Reply:
ok great. this works cause im currently documenting actor flags, and theres this one 
`#define ACTOR_FLAG_CAN_HOLD_SWITCHES (1 << 26)`
given to ruto, armos etc